force-app/main/default/objects/Contact_Association__c/fields/Contact__c.field-meta.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Contact__c</fullName>
+    <description>Contact related to the Job Application</description>
+    <externalId>false</externalId>
+    <label>Contact</label>
+    <referenceTo>Contact</referenceTo>
+    <relationshipLabel>ContactAssociations</relationshipLabel>
+    <relationshipName>ContactAssociations</relationshipName>
+    <relationshipOrder>0</relationshipOrder>
+    <reparentableMasterDetail>false</reparentableMasterDetail>
+    <trackTrending>false</trackTrending>
```

Comment:
If we do not want to delete historical data, we need to allow for re-parenting

Reply:
updated!